packages/components/src/LightBox/LightBox.tsx

```diff
@@ -92,4 +138,86 @@ export function LightBox({
   function handleMoveNext() {
     setCurrentImageIndex((currentImageIndex + 1) % images.length);
   }
+
+  function handleImageWrapperClick(
+    event: React.MouseEvent<HTMLDivElement, MouseEvent>,
```

Comment:
Since we're destructuring the other ones like ReactNode instead of React.ReactNode, mind following that pattern for `MouseEvent`?

Reply:
I think for this case `React.MouseEvent` is different than `MouseEvent`. `MouseEvent` refers to the DOM, while the other is React-specific. I tried to change its import from React, but then I get an error on `onEndDrag` because it's referencing DOM's MouseEvent.

Reply:
I mean like import it above 

```
import React, { MouseEvent, ReactNode, useEffect, useState } from "react";
```

Reply:
I did, but than I get type errors